opae.spec.in

```diff
@@ -151,7 +151,9 @@ ldconfig
 @CMAKE_INSTALL_PREFIX@/bin/bist_app.py
 @CMAKE_INSTALL_PREFIX@/bin/bist_common.py
 @CMAKE_INSTALL_PREFIX@/bin/bist_dma.py
+@CMAKE_INSTALL_PREFIX@/bin/bist_def.py
```

Comment:
The only concern I have with this pull is including bist_def.py.  We had it on master for a while, and it broke the RPM creation process.  Could you verify that the RPM can still be created on this branch?

Reply:
tools: merge fpgabist from internal opae
https://github.com/OPAE/opae-sdk/pull/1283

bist_def.py is include in pull request. RPM creation process passes after FPGABIST PR.